### ISSUE TITLE
Hololive Resistance logo

### ISSUE THREAD

**Issue by holo-dango** [HL-Resistance/hlresistance-preview.github.io#12]
Does the Hololive Resistance group have its own logo? If so, I suggest using that instead of the official Hololive logo. It would better represent the community and prevent unintended copyright issues from using another company's branding.

**Comment by GoldElysium** [comment]
I think we should replace the background as well to something where we have the correct rights on

**Comment by Aeoraa** [comment]
I agree that the background and logo should be replaced with something we have rights on. EK template test page logo with the word "resistance" instead of "production" still look similar to the official Hololive logo, perhaps a redesign of it will be better?

**Comment by GoldElysium** [comment]
Logo and background will be created by the art team

**Comment by Ekwok95** [comment]
We won't release the website until we're sure there wont be any issues with the content we put up.

**Comment by GoldElysium** [comment]
Assigning Ek so it won't show as help needed anymore
